java/arcs/core/data/proto/HandleProtoDecoder.kt

```diff
@@ -47,11 +47,11 @@ fun List<HandleProto.Capability>.decode() = Capabilities(
  *
  * If a type is not set in the [HandleProto], it is initialized to a newly created TypeVariable.
  */
-fun HandleProto.decode() = Handle(
+fun HandleProto.decode(knownHandles: Map<String, Handle> = mutableMapOf()) = Handle(
```

Comment:
No need to have the default value be a MutableMap here.

```suggestion
fun HandleProto.decode(knownHandles: Map<String, Handle> = emptyMap()) = Handle(
```